### ISSUE TITLE
OSX build not working properly

### ISSUE THREAD

**Issue by albertogasparin** [metal3d/idok#5]
I have downloaded the build 20140910-1 darwin but it is partially working on Yosemite. It is possible to send an url to Kodi, but the streaming of a local file fails:

```
idok -target=KODI_IP /my/local/file.mp4
```

Outputs: `Unable to get local ip`.
So I have downloaded idok source code and I have rebuilt the project. Magically, the same code (`utils/ip.go`) works! I don't know why...
Anyhow, if you are interested I can send you my working version of idok for OSX Yosemite (I am also planning a simple GUI). 
Thanks for this amazing utility!

**Comment by albertogasparin** [comment]
Moreover, another issue I've found with the build 20140910-1 is that the program does not quit after Kodi response. For instance, running this command:

```
./idok -target=KODI_IP https://streaming.jamendo.com/JamRock
```

Will successfully start the playback on Kodi and it prints `{"id":1,"jsonrpc":"2.0","result":"OK"}` on the shell.
However, the program does not quit and I have to manually press `CTRL+c` to force quit it.
This issue disappears with my Yosemite build (and no code changes).

**Comment by metal3d** [comment]
Hi,

Thaks for the report. That's very complicated for me to test my darwin build because I don't have MacOS. I'm using built-in compilation.

I guess you downloaded the last code version that have some modifications. Right now, I didn't made any release (I will do it this week, I need time to test last changes). So maybe the problem disapeared with my last fixes, and not because a compilation problem.

Or maybe I'm wrong, and I need to fix the compilation process.

I keep this issue opened and I will try to build new release. Then, If you've got time, maybe you can confirm my hypothesis :)

Thanks a lot

**Comment by metal3d** [comment]
For the second bug, I already had this kind of behaviour. I will try it. But please, can you create another issue to not block the first one ? Thanks again :)

**Comment by albertogasparin** [comment]
Yes, exactly. I have filed both issues here because by compiling "master" on my Mac Idok works perfectly. So I think they are both related to the build process.

**Comment by metal3d** [comment]
If you used master branch so you have the fixed version and probably the "build process" is not in cause. I will provide a new release from the master branch and you will be able, if you've got time, to test it. If the new release has the same problem, so the build process is the problem, if no so this is a fix I introduced after the release you've used.
I will try to find time tomorow to do that.

Thanks :)

**Comment by metal3d** [comment]
New release made - you can check :)
PS: your contribution is notified in AUTHORS
